.github/workflows/on-prerelease.yml

```diff
@@ -158,7 +158,7 @@ jobs:
       contents: write
     steps:
       - name: Download the SBOM
-        uses: Ed-Fi-Alliance-OSS/slsa-github-generator/.github/actions/secure-download-artifact@main
+        uses: Ed-Fi-Alliance-OSS/slsa-github-generator/.github/actions/secure-download-artifact@f4c84deb58e41f45b8f67aec54e5a8a4be9e1f4c # main
```

Comment:
@dfernandez-gap please revert this one. There's a weird thing about this specific workflow where we can't use a hash to pin the dependency. We must use `main`. You wouldn't have known that of course, so you were doing the right thing by trying to change this. In the repository security warnings I will dismiss this specific warning.